src/Notification/docs/index.story.js

```diff
@@ -21,14 +21,27 @@ import typesReadme from './types.readme.md';
 const exampleChildren = [
   {
     label: 'Just text',
+    value: [<Notification.TextLabel>Notification text</Notification.TextLabel>],
```

Comment:
```suggestion
    value: <Notification.TextLabel>Notification text</Notification.TextLabel>,
```

there's a bug hidden somewhere in `wix-storybook-utils` which that `[object Object]` initially instead of `label`. As a quick workaround, which in this case works, passing value without array shows correct label